developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,222 @@ You can request additional properties by sending a request body with the :code:`
+|                               | |                                               | |     ``<d:locktype><d:write /></d:locktype>``                           |
+|                               | |                                               | | ``</d:lockentry>``                                                     |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:id />                     | | The fileid namespaced by the instance id.     | ``00000007oc9l3j5ur4db``                                                 |
+|                               | | Globally unique.                              |                                                                          |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:fileid />                 | The unique id for the file within the instance. | ``7``                                                                    |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:downloadURL />            | | A URL to directly download the file from a    |                                                                          |
+|                               | | storage. No storage implements that yet.      |                                                                          |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:permissions />            | | The permissions that the user has over the    | | ``S``: Shared                                                          |
+|                               | | file. The value is a string containing        | | ``R``: Shareable                                                       |
+|                               | | letters for all available permissions.        | | ``M``: Mounted                                                         |
+|                               |                                                 | | ``G``: Readable                                                        |
+|                               |                                                 | | ``D``: Deletable                                                       |
+|                               |                                                 | | ``NV``: Updateable, Renameable, Moveable                               |
+|                               |                                                 | | ``W``: Updateable (file)                                               |
```

Comment:
```suggestion
|                               |                                                 | | ``W``: Writable (file)                                               |
```

Reply:
Or is this copied from some other documentation?

Reply:
I kept the same nomenclature than https://github.com/nextcloud/server/blob/artonge/fix/do_not_allow_to_show_pwd_in_sharing/extra_apps/firstrunwizard/vendor/nextcloud/ocp/OCP/Files/DavUtil.php#L75-L81

Reply:
I can't see that file :thinking:

Reply:
Indeed, the link is broken, here is a working one: https://github.com/nextcloud/server/blob/artonge/fix/do_not_allow_to_show_pwd_in_sharing/lib/public/Files/DavUtil.php#L75-L81